pkg/storaged/pages.jsx

```diff
@@ -816,14 +816,14 @@ export const StoragePage = ({ location, plot_state }) => {
     return (
         <Page id="storage">
             { parent_crumbs.length > 0 &&
-            <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
+            <PageBreadcrumb stickyOnBreakpoint={client.in_anaconda_mode() ? {} : { default: "top" }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19876-20240124-094035-6c2858a4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)